### ISSUE TITLE
UiSelect filter function need third parameter

### ISSUE THREAD

**Issue by projct1** [JosephusPaye/Keen-UI#229]
This parameter must be default, i.e:
```javascript
filter(item, query, default) {
    return item.id == query || default(item, query);
}
```

**Comment by JosephusPaye** [comment]
Added in https://github.com/JosephusPaye/Keen-UI/commit/3a5ae7e940bb7914b8112e92c424aa8124d549a3.

**Comment by JosephusPaye** [comment]
Released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
